src/ripple/rpc/impl/RPCHandler.cpp

```diff
@@ -157,6 +157,12 @@ error_code_i fillHandler (Context& context,
         return rpcNO_NETWORK;
     }
 
+    if (context.app.getOPs().isAmendmentBlocked() &&
+        handler->condition_ & NEEDS_CURRENT_LEDGER)
+    {
+        return rpcAMENDMENT_BLOCKED;
+    }
+
     if (!context.app.config().standalone() &&
         handler->condition_ & NEEDS_CURRENT_LEDGER)
```

Comment:
yeah, I wondered that too and had a hard time deciding since, as you point out, there aren't any methods currently using it. I think I'll add it to the check as you suggest.